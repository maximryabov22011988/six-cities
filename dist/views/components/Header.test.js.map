{"version":3,"file":"Header.test.js","sourceRoot":"","sources":["../../../src/views/components/Header.test.tsx"],"names":[],"mappings":";;AAAA,+BAA+B;AAC/B,gDAAgD;AAEhD,qCAA8B;AAE9B,EAAE,CAAC,+CAA+C,EAAE,GAAG,EAAE;IACvD,MAAM,IAAI,GAAG,QAAQ;SAClB,MAAM,CACL,oBAAC,gBAAM,IACL,IAAI,EAAE,IAAI,EACV,QAAQ,EAAE,IAAI,GACd,CACH,CAAC,MAAM,EAAE,CAAC;IAEb,MAAM,CAAC,IAAI,CAAC,CAAC,eAAe,EAAE,CAAC;AACjC,CAAC,CAAC,CAAC"}