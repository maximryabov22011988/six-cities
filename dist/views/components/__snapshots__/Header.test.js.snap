// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly with mandatory props 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="header__wrapper"
    >
      <div
        className="header__left"
      />
      <nav
        className="header__nav"
      >
        <ul
          className="header__nav-list"
        />
      </nav>
    </div>
  </div>
</header>
`;
